app/models/katello/docker_tag.rb

```diff
@@ -10,6 +10,8 @@ class DockerTag < Katello::Model
       :complete_value => true, :only_explicit => false
     scoped_search :relation => :docker_manifest, :on => :digest, :rename => :digest,
       :complete_value => false, :only_explicit => true
+    scoped_search :relation => :docker_manifest, :on => :schema_version, :rename => :schema_version,
+      :complete_value => false, :only_explicit => true
```

Comment:
we have quite a few:

$ grep 'search_for' test/ -R  | awk '{print $1}' |uniq
test/models/pool_test.rb:
test/models/content_view_filter_test.rb:
test/models/host/content_facet_test.rb:
test/models/host/subscription_facet_test.rb:
test/models/content_view_test.rb:
test/models/activation_key_test.rb:
test/models/package_group_test.rb:
test/models/repository_test.rb:
test/models/erratum_test.rb:
test/models/concerns/host_managed_extensions_test.rb:
test/models/concerns/environment_extensions_test.rb:
test/models/product_test.rb:
test/models/content_view_version_test.rb:
test/models/content_view_puppet_module_test.rb:
test/models/host_collection_test.rb:


and the goal isn't to test scoped search, it to test your scoped search definition.  Similar to AR association definitions, you can easily define them (or modify them) in a way that makes them not work properly, so testing is still important IMHO